libcaf_io/src/basp_broker.cpp

```diff
@@ -144,17 +145,17 @@ behavior basp_broker::make_behavior() {
     [=](forward_atom, strong_actor_ptr& src,
         const std::vector<strong_actor_ptr>& fwd_stack, strong_actor_ptr& dest,
         message_id mid, const message& msg) {
-      CAF_LOG_TRACE(CAF_ARG(src) << CAF_ARG(dest)
-                    << CAF_ARG(mid) << CAF_ARG(msg));
+      CAF_LOG_TRACE(CAF_ARG(src)
+                    << CAF_ARG(dest) << CAF_ARG(mid) << CAF_ARG(msg));
       if (!dest || system().node() == dest->node()) {
-        CAF_LOG_WARNING("cannot forward to invalid or local actor:"
-                        << CAF_ARG(dest));
+        CAF_LOG_WARNING(
+          "cannot forward to invalid or local actor:" << CAF_ARG(dest));
```

Comment:
I'm not sure about this one either. Doesn't really matter though.

Reply:
I've broken up the string literal to get a nicer formatting.